default/space_monsters.txt

```diff
@@ -397,7 +397,7 @@ ShipDesign
     hull = "SH_BLOATED_JUGGERNAUT_BODY"
     parts = [
         ""
-        "SP_BIOTERM"
```

Comment:
I'm not exactly sure what you're referring to... do you mean if "SP_BIOTERM" is still in use? If that's what you want to know, then yes. Why?